docs/templates/index.md

```diff
@@ -0,0 +1,199 @@
+# Templates 
+
+A template is a collection of configuration parameters that is used to launch a Triton instance. Templates are immutable,
+therefore, once they are created, they cannot be changed, therefore a new template needs created. A template is made up
+as follows:
+
+| Field             | Type              |  Description                                                                  |
+|----------         |-------------      |------                                                                         |
+| ID                | string            | The UUID of the Template.                                                     |
+| TemplateName      | string            | The name of the Template.                                                     |
+| AccountID         | string            | The AccountID the Template is associated to.                                  | 
+| Package           | string            | The ID of the Package to use when launching a Template.                       | 
+| ImageID           | string            | The ID of the Package to use when launching a Template.                       | 
+| FirewallEnabled   | bool              | Enable or disable the Firewall on the instances launched. Default is `false`. | 
+| Networks          | []string          | A list of network IDs to attach to the instances launched.                    | 
+| UserData          | string            | Data to be copied to the instances on boot.                                   | 
+| MetaData          | map[string]string | A mapping of metadata to apply to the instances launched.                     | 
+| Tags              | map[string]string | A mapping of tags to apply to the instances launched.                         |  
+
+### GET /v1/tsg/templates
+
+To list all of the templates associated with a specific Triton account, send a `GET` request to `/v1/tsg/templates` with
+the request headers detailed below.
+
+#### Example Request
+
+```
+curl -X GET "https://tsg.us-sw-1.svc.joyent.zone/v1/tsg/templates"
+```
+
+
+#### Request Headers 
+
+```
+Authorization: Signature keyId="/test-user-name/keys/aa:bb:cc:dd:9c:54:e9:78:3f:80:0d:ba:6b:c6:ff:44",algorithm="rsa-sha1",headers="date",signature="..."
```

Comment:
I think you can drop `headers="date"` from the header

Reply:
We might be able to but that’s currently baked into the Triton-go client.